// @clerk/themes/dist/themes/dark.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var dark_exports = {};
__export(dark_exports, {
  dark: () => dark
});
module.exports = __toCommonJS(dark_exports);
var import_createTheme = require("../createTheme");
const dark = (0, import_createTheme.experimental_createTheme)({
  name: "dark",
  variables: {
    colorBackground: "#212126",
    colorNeutral: "white",
    colorPrimary: "#ffffff",
    colorPrimaryForeground: "black",
    colorForeground: "white",
    colorInputForeground: "white",
    colorInput: "#26262B"
  },
  elements: {
    providerIcon__apple: { filter: "invert(1)" },
    providerIcon__github: { filter: "invert(1)" },
    providerIcon__okx_wallet: { filter: "invert(1)" },
    providerIcon__vercel: { filter: "invert(1)" },
    activeDeviceIcon: {
      "--cl-chassis-bottom": "#d2d2d2",
      "--cl-chassis-back": "#e6e6e6",
      "--cl-chassis-screen": "#e6e6e6",
      "--cl-screen": "#111111"
    }
  }
});
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  dark
});
