// @clerk/themes/dist/themes/shadcn.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var shadcn_exports = {};
__export(shadcn_exports, {
  shadcn: () => shadcn
});
module.exports = __toCommonJS(shadcn_exports);
var import_createTheme = require("../createTheme");
const shadcn = (0, import_createTheme.experimental_createTheme)({
  name: "shadcn",
  cssLayerName: "components",
  variables: {
    colorBackground: "var(--card)",
    colorDanger: "var(--destructive)",
    colorForeground: "var(--card-foreground)",
    colorInput: "var(--input)",
    colorInputForeground: "var(--card-foreground)",
    colorModalBackdrop: "var(--color-black)",
    colorMuted: "var(--muted)",
    colorMutedForeground: "var(--muted-foreground)",
    colorNeutral: "var(--foreground)",
    colorPrimary: "var(--primary)",
    colorPrimaryForeground: "var(--primary-foreground)",
    colorRing: "var(--ring)",
    fontWeight: {
      normal: "var(--font-weight-normal)",
      medium: "var(--font-weight-medium)",
      semibold: "var(--font-weight-semibold)",
      bold: "var(--font-weight-semibold)"
    }
  },
  elements: {
    input: "bg-transparent dark:bg-input/30",
    cardBox: "shadow-sm border",
    popoverBox: "shadow-sm border",
    button: {
      '&[data-variant="solid"]::after': {
        display: "none"
      }
    },
    providerIcon__apple: "dark:invert",
    providerIcon__github: "dark:invert",
    providerIcon__okx_wallet: "dark:invert",
    providerIcon__vercel: "dark:invert"
  }
});
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  shadcn
});
