// @clerk/themes/dist/themes/shadesOfPurple.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var shadesOfPurple_exports = {};
__export(shadesOfPurple_exports, {
  shadesOfPurple: () => shadesOfPurple
});
module.exports = __toCommonJS(shadesOfPurple_exports);
var import_createTheme = require("../createTheme");
var import_dark = require("./dark");
const shadesOfPurple = (0, import_createTheme.experimental_createTheme)({
  name: "shadesOfPurple",
  baseTheme: import_dark.dark,
  variables: {
    colorBackground: "#3f3c77",
    colorPrimary: "#f8d80d",
    colorPrimaryForeground: "#38375f",
    colorInputForeground: "#a1fdfe",
    colorShimmer: "rgba(161,253,254,0.36)"
  }
});
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  shadesOfPurple
});
